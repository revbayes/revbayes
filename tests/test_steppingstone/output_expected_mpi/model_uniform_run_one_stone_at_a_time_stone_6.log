Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-22174.88	-22163.92	-10.95983	0.7201563	0.027745	0.0440474	0.0401723	0.0350505	0.122838	0.111306	0.320722	0.106703	0.157936	0.123537	0.0701289	0.206148	0.228481	0.352007	0.097217	0.123188	0.180942	0.0721553	0.206553	0.167098	0.141867	0.110489	0.958078	0.022268	0.0901945	0.00914465	0.0329921	0.0216615	0.108542	0.280609	0.246472	0.0265865	0.135469	0.0559058	0.076358	0.0304166	0.103426	0.0238457	0.0210998	0.0442508	0.0250369	0.0160493	0.339674	0.0316516	0.34726	0.0766816	0.0231767	0.498604	0.0226262	0.364695	0.332125	0.0877299	0.21545	0.319079	5.714411
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
10	-22361.29	-22357.97	-3.325717	0.5908261	0.0575031	0.0177932	0.014153	0.117167	0.0659327	0.149904	0.153423	0.159832	0.175014	0.0617428	0.230814	0.317952	0.235348	0.264015	0.0900768	0.216438	0.0818591	0.101519	0.0824199	0.223285	0.0663535	0.178507	0.232442	0.0183073	0.0591057	0.0100072	0.107046	0.0515048	0.176378	0.226237	0.178888	0.0183469	0.0593894	0.0876677	0.0492837	0.0286206	0.0874773	0.0286765	0.024043	0.0360957	0.0254826	0.0334244	0.30704	0.0824648	0.364702	0.120504	0.0266322	0.39874	0.00695696	0.290335	0.32876	0.096318	0.284587	0.4134076	4.906518
