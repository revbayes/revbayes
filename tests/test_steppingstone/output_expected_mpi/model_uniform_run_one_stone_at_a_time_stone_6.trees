Iteration	Posterior	Likelihood	Prior	psi
0	-22533.69	-22530.29	-3.401854	((((Macaca_mulatta[&index=13]:0.254538,Pan_paniscus[&index=17]:0.015950)[&index=30]:0.145716,((Chlorocebus_aethiops[&index=5]:0.030181,Colobus_guereza[&index=6]:0.110097)[&index=25]:0.067687,Hylobates_lar[&index=9]:0.117620)[&index=42]:0.034406)[&index=32]:0.152619,(Cebus_albifrons[&index=3]:0.157467,((Callicebus_donacophilus[&index=2]:0.115133,Aotus_trivirgatus[&index=1]:0.031705)[&index=26]:0.004566,Saimiri_sciureus[&index=20]:0.044592)[&index=33]:0.106344)[&index=43]:0.648234)[&index=28]:0.032471,((Otolemur_crassicaudatus[&index=16]:0.127795,((Perodicticus_potto[&index=18]:0.022096,(Nycticebus_coucang[&index=15]:0.088301,(Daubentonia_madagascariensis[&index=7]:0.226939,((Lepilemur_hubbardorum[&index=11]:0.248753,Varecia_variegata_variegata[&index=22]:0.118824)[&index=35]:0.018761,(Microcebus_murinus[&index=14]:0.266509,((Lemur_catta[&index=10]:0.017401,Tarsius_syrichta[&index=21]:0.221920)[&index=39]:0.181843,(Cheirogaleus_major[&index=4]:0.148591,Propithecus_coquereli[&index=19]:0.076440)[&index=38]:0.068529)[&index=34]:0.046246)[&index=27]:0.049060)[&index=40]:0.047080)[&index=29]:0.089586)[&index=41]:0.043759)[&index=36]:0.101796,Loris_tardigradus[&index=12]:0.093239)[&index=31]:0.022725)[&index=24]:0.023243,Galago_senegalensis[&index=8]:0.336547)[&index=37]:0.051679,Galeopterus_variegatus[&index=23]:0.175728)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
10	-22880.7	-22876.61	-4.090426	((((((Chlorocebus_aethiops[&index=5]:0.021302,Colobus_guereza[&index=6]:0.103717)[&index=25]:0.099836,(Hylobates_lar[&index=9]:0.079498,(Pan_paniscus[&index=17]:0.006276,Macaca_mulatta[&index=13]:0.228757)[&index=42]:0.014887)[&index=32]:0.099781)[&index=30]:0.135107,(Cebus_albifrons[&index=3]:0.232076,(Aotus_trivirgatus[&index=1]:0.023699,Callicebus_donacophilus[&index=2]:0.165495)[&index=26]:0.006781)[&index=33]:0.070597)[&index=28]:0.047575,Saimiri_sciureus[&index=20]:0.105914)[&index=43]:0.527500,(Otolemur_crassicaudatus[&index=16]:0.227345,Perodicticus_potto[&index=18]:0.094955)[&index=24]:0.018763)[&index=36]:0.061454,Galeopterus_variegatus[&index=23]:0.334769,((Galago_senegalensis[&index=8]:0.281425,(Daubentonia_madagascariensis[&index=7]:0.072502,(Cheirogaleus_major[&index=4]:0.112579,((Lepilemur_hubbardorum[&index=11]:0.249196,Varecia_variegata_variegata[&index=22]:0.101327)[&index=27]:0.133048,((Microcebus_murinus[&index=14]:0.278293,Tarsius_syrichta[&index=21]:0.086460)[&index=34]:0.190522,(Propithecus_coquereli[&index=19]:0.038559,Lemur_catta[&index=10]:0.077294)[&index=38]:0.091840)[&index=39]:0.075343)[&index=40]:0.037313)[&index=35]:0.035206)[&index=29]:0.038357)[&index=37]:0.152105,(Nycticebus_coucang[&index=15]:0.068078,Loris_tardigradus[&index=12]:0.091867)[&index=41]:0.134949)[&index=31]:0.006394)[&index=44];
