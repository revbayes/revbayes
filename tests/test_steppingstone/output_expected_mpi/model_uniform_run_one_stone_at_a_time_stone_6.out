state	power	likelihood
4	0.0173415	-22221.3
5	0.0173415	-22066.9
6	0.0173415	-22153.7
7	0.0173415	-22226.5
8	0.0173415	-22390.8
9	0.0173415	-22336.6
10	0.0173415	-22358
